server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -196,14 +196,25 @@ def validate_openshift_parameters(deployment)
           deployment.errors[:openshift_storage_size] << _("OpenShift deployments must have a storage size greater than zero")
         end
 
-        validate_storage_share(deployment, deployment.openshift_storage_host, deployment.openshift_export_path, -1, -1)
+        if deployment.openshift_storage_host.empty?
```

Comment:
They're displayed in a list to the user on the review page and we usually need to list all errors for all fields, so it would be preferable.